ush/python/pygfs/task/globus_hpss.py

```diff
@@ -0,0 +1,494 @@
+#!/usr/bin/env python3
+
+import os
+import logging
+import shutil
+from time import sleep
+from typing import Any, Dict, List
+import re
+import copy
+from datetime import datetime, timezone
+
+from wxflow import AttrDict, Task, to_YMDH, logit, parse_yaml, Jinja, which, ProcessError, to_datetime
+
+logger = logging.getLogger(__name__.split('.')[-1])
+logging.basicConfig(encoding='utf-8', level=logging.DEBUG, format='%(asctime)s %(message)s')
+
+
+class GlobusHpss(Task):
+    """Task to send tarballs (created by the archive task) to HPSS via Globus
+       NOTE: For this to work, an entry in ~/.ssh/config titled "niagara" must
+             be present.  If it is not, then see the wiki on how to set it up.
```

Comment:
I don't think I like this. Some users' (mine included) ssh config is not located at `~/.ssh/config`. Others probably don't even know what it is. On some machines it's not even allowed. Can we just use the correct hostname and set the username somewhere? Could even prompt for it at setup time if necessary.

Reply:
Deleted.